examples/games/game_menu.rs

```diff
@@ -789,8 +789,8 @@ mod menu {
         mut menu_state: ResMut<NextState<MenuState>>,
         mut game_state: ResMut<NextState<GameState>>,
     ) {
-        for event in click_events.read() {
-            if let Ok(menu_button) = menu_buttons.get(event.0) {
+        for &Click(clicked_entity) in click_events.read() {
```

Comment:
In my opinion it's longer to type it like this. I probably would keep `for event in ...` and change that one occurrence that did that originally.

Reply:
fair, i have no preference